enhancements/upgrade-bundle.md

```diff
@@ -0,0 +1,138 @@
+---
+title: upgrade-bundle
+authors:
+  - "@rashmigottipati"
+reviewers:
+  - TBD
+approvers:
+  - TBD
+creation-date: 2020-06-11
+last-updated: 2020-06-11
+
+---
+
+# Upgrade Bundle
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+
+## Open Questions
+
+1. How do we handle multiple subscriptions? 
```

Comment:
This question is critical to answer because this command needs smart detection logic for whether we need to create a new `Subscription` (same channel? same starting CSV? same operator?) or error out.

@njhale @ecordell are `Subscription`'s patch-able, ex. can the starting CSV or channel be updated?

Reply:
I don't have a specific use-case for multiple subscriptions. Maybe @estroz does.

Reply:
I updated the PR to include these details for that open question.